Automattic/wp-calypso
# Hosting Command Palette: Create an initial prototype for Calypso /sites with default commands - version 2

Related to: https://github.com/Automattic/dotcom-forge/issues/4411 and 

## Proposed Changes

This PR adds command pallette to the `/sites` page. The command pallette opens when you press the shortcut `cmd + k` on Mac or `ctrl + k` on Windows

## Testing Instructions

* Navigate tp `/sites`
* Press `cmd + k` on Mac or `ctrl + k` on Windows
* Confirm that you see the new command pallette appearing
* Confirm that you can select different options in the command pallette and you are brought to the correct place when you click on a specific option: 

<img width="1176" alt="Screenshot 2023-11-03 at 4 16 54 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/6169d30f-7d21-4c29-b260-ce2b054df59e">


## Pre-merge Checklist
